lib/stdlib/doc/src/sets.xml

```diff
@@ -174,8 +174,13 @@
       <name name="is_set" arity="1" since=""/>
       <fsummary>Test for a <c>Set</c>.</fsummary>
       <desc>
-        <p>Returns <c>true</c> if <c><anno>Set</anno></c> is a set of
-          elements, otherwise <c>false</c>.</p>
+        <p>Returns <c>true</c> if <c><anno>Set</anno></c> appears to
+        be a set of elements, otherwise <c>false</c>.  Note that the
+        test is shallow and will return <c>true</c> for any term that
+        coincides with the possible representations of a set.  See
+        also note on <seeguide
+        marker="system/reference_manual:data_types#no_user_types">data
```

Comment:
(general comment from learners perspective) linked section is very useful for understanding what **is not** Erlang type and what `set` is from environment perspective. maybe consider adding the link in general description (beginning) of each reference manuals as well (sets, sofs etc).

For example for sets, maybe a link could fit somewhere near to fragment below  (unless we don't want to state representation is an undefined composite type - however I think that with your additions we kind of indirectly state it already).
> Sets are collections of elements with no duplicate elements. The representation of a set is undefined.

I think "composite type" is a really nice term, and maybe should be promoted.

Reply:
Good idea.  Now done.